2-mandatory/3-financial-times.js

```diff
@@ -14,7 +21,10 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+  tilteWithFewest = allArticleTitles.filter((item) => item.length <= 65);
```

Comment:
You're pretty close here, what you're doing is you're checking against the length of characters instead of the number of words. Words are split up by a space, some words may be 2 characters long and some may be up to 45 characters long. (The longest word in the English dictionary is "Pneumonoultramicroscopicsilicovolcanoconiosis" so if you imagine the title is that word twice, the word count is 2 but the character count is 91 - it'd be the shortest word count but would not pass this check)

Reply:
Yes I understand my mistake.
Thanks I will change it